### PR TITLE
Add Wishlist Page

### PR DESCRIPTION
**Pull request by varun13169**
Add Wishlist Page

### CODE REVIEW

File: pages/wishlistPage/wishlistPage.html [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+                        <svg xmlns="http://www.w3.org/2000/svg"
+                            class="dui-inp-txt__icon-content dui-inp-txt__icon-content--search icon icon-tabler icon-tabler-search"
+                            width="56" height="56" viewBox="0 0 24 24" stroke-width="1.5" stroke="#059669" fill="none"
+                            stroke-linecap="round" stroke-linejoin="round">
+                            <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+                            <circle cx="10" cy="10" r="7" />
+                            <line x1="21" y1="21" x2="15" y2="15" />
+                        </svg>
+                    </i>
+                    <input
```
Comment (itsmebipul00) [inline pages/wishlistPage/wishlistPage.html]: a label tag would look nice here

File: pages/wishlistPage/wishlistPage.html [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+                        <!-- Badge Component Starts -- with Text -->
+                        <div class="dui-badge">
+                            <div class="dui-util-txt-sm dui-util-fw-reg">
+                                SALE
+                            </div>
+                        </div>
+                        <!-- Badge Component Ends -- with Text -->
+
+                        <img class="dui-card__img dui-card__img--hr-prod"
+                            src="/assets/potted-plant-gda1ef23cd_640.png" />
```
Comment (itsmebipul00) [inline pages/wishlistPage/wishlistPage.html:263]: pngs files are the heaviest of all image file types

File: pages/wishlistPage/wishlistPage.html [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+
+                        <div class="dui-card__info dui-util-spc-pad-0_8rem-xs">
+                            <p class="dui-card__secondary-text">Amazing Indoor Plant</p>
+                            <h3 class="dui-card__primary-text dui-util-fw-blk">$2000 <s
```
Comment (itsmebipul00) [inline pages/wishlistPage/wishlistPage.html:371]: for semantic purposes you could use a p tag with a higher font size and not a heading tag for $2000

**Review by itsmebipul00 — APPROVED** [review]
Good to merge, if possible with changes!!

File: pages/wishlistPage/wishlistPage.html (hunk 2 above)
Comment (varun13169) [reply pages/wishlistPage/wishlistPage.html:263]: Right now these are just placeholders for the UI and not actual images.

will be using **srcset** while optimizing for performance.

Comment (itsmebipul00) [reply pages/wishlistPage/wishlistPage.html:263]: Okay nice!

File: pages/wishlistPage/wishlistPage.html (hunk 3 above)
Comment (varun13169) [reply pages/wishlistPage/wishlistPage.html:371]: I did it on purpose to semantically emphasize the price.

File: pages/wishlistPage/wishlistPage.html (hunk 1 above)
Comment (varun13169) [reply pages/wishlistPage/wishlistPage.html]: thanks for the catch, made the change here will be making the same change site-wide as well.

File: pages/wishlistPage/wishlistPage.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.main-content_wishlist-holder {
+    display: grid;
+    gap: 2rem;
+    grid-template-columns: 1fr 1fr 1fr;
```
Comment (iRohitGaur) [inline pages/wishlistPage/wishlistPage.css:26]: suggestion: you can use "repeat(3, 1fr)" here

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: pages/wishlistPage/wishlistPage.css (hunk 4 above)
Comment (varun13169) [reply pages/wishlistPage/wishlistPage.css:26]: I'll take this as part of refactoring, due to the learning curve right now I am writing things explicitly.

**Pull request merged**